scripts/Controller.ice

```diff
@@ -8,6 +8,7 @@ sequence<string> StringSeq;
 
 class Config
 {
+    ["swift:identifier:`protocol`"]
```

Comment:
I would only use keyword-in-ticks in actual tests.

Here, I'd remap it to `transportProtocol`.

Reply:
Alright. Done.